Formula/gst-plugins-rs.rb

```diff
@@ -20,18 +24,53 @@ class GstPluginsRs < Formula
   depends_on "cargo-c" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
+  depends_on "python@3.11" => :build # needs python with tomllib
   depends_on "rust" => :build
   depends_on "dav1d"
+  depends_on "gst-plugins-bad" # for webrtc
   depends_on "gst-plugins-base"
   depends_on "gstreamer"
   depends_on "gtk4"
   depends_on "pango" # for closedcaption
 
-  # commit ref, https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs/-/commit/ea98a0b5964cd196abbb48c621969a8ef33eb157
-  # remove in next release
-  patch :DATA
+  # Fix building `libwebp-sys2` crate on Intel macOS (SSE4.1 support).
+  # The resource version matches version picked by Cargo dependency resolution.
+  # We use git URL rather than generated source tarball to get submodule.
+  # The crate could also be used, but it requires manual extraction as brew
+  # cannot handle .crate extension and this means we cannot use patch DSL.
+  # TODO: Remove when upstream PR is approved and available in release.
+  on_macos do
+    on_intel do
+      resource "libwebp-sys2" do
+        url "https://github.com/qnighy/libwebp-sys2-rs.git",
+            tag:      "v0.1.4",
+            revision: "3ecdb2e4898820d46436a76e1cfee75ec14a881a"
+
+        # PR ref: https://github.com/qnighy/libwebp-sys2-rs/pull/13
+        patch do
+          url "https://github.com/qnighy/libwebp-sys2-rs/commit/eddb6e2d1ef88b1b1a68d7709733e7dff81f6659.patch?full_index=1"
+          sha256 "0bee66cf8e29880719c64d54be0e8199da91c8d4aa9fce13e26b0cc024371e4d"
+        end
+      end
+    end
+  end
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    # TODO: Use `webp` dependency on macOS when supported.
+    # Issue ref: https://github.com/qnighy/libwebp-sys2-rs/issues/4
+    depends_on "webp"
```

Comment:
Need to check if library is linked to make sure build picked up Homebrew's copy.